axelrod/plot.py

```diff
@@ -279,6 +279,7 @@ def save_all_plots(self, prefix="axelrod", title_prefix="axelrod",
             f = getattr(self, method)(title="{} - {}".format(title_prefix,
                                                              name))
             f.savefig("{}_{}.{}".format(prefix, method, filetype))
+            plt.close(f)
```

Comment:
THANK YOU! (This error message has been like a really annoying pebble in a shoe on a long walk...)